package.json

```diff
@@ -267,7 +267,7 @@
 		"playground:build": "npm run storybook:build",
 		"playground:dev": "echo \"Please use storybook:dev instead.\"",
 		"env": "wp-scripts env",
-		"wp-env": "packages/env/bin/wp-env"
+		"wp-env": "wp-env"
```

Comment:
Curious, does Node resolve the local version from packages with this or the global version?